hana/lib/cql-functions.js

```diff
@@ -24,10 +24,39 @@ const StandardFunctions = {
   contains: (...args) => args.length > 2 ? `CONTAINS(${args})` : `(CASE WHEN coalesce(locate(${args}),0)>0 THEN TRUE ELSE FALSE END)`,
   concat: (...args) => `(${args.map(a => (a.xpr ? `(${a})` : a)).join(' || ')})`,
   search: function (ref, arg) {
+    const csnElements = ref.element ? [ref] : [...ref.list]
+    let fuzzyString
+
+    // default config
+    const fuzzyIndex = cds.env.hana?.fuzzy || 0.7
+
+    // if column specific value is provided, the configuration has to be defined on column level
+    if (csnElements.some(e => e.element?.['@Search.ranking'] || e.element?.['@Search.fuzzinessThreshold'])) {
+      const cols = csnElements.map(e => {
+        // REVISIT: How to do quoting?
+        let col = `${e.ref.join('.')} FUZZY`
```

Comment:
there is a `quote` function in `cqn2sql` which takes care of quotation, if necessary. Bind the `this` of the `cqn2sql` instance and use it here, I would say.

Reply:
Bob so far uses a different technique by changing the toString method. That does not suite my use case but I don't know the implications of changing it. I propose to discuss later